app/jobs/import_data_from_rapids_job.rb

```diff
@@ -0,0 +1,51 @@
+class ImportDataFromRAPIDSJob < ApplicationJob
+  queue_as :default
+
+  PER_PAGE_SIZE = 50
+
+  def perform
+    service = RAPIDS::API.call
+    start_index = 1
+
+    loop do
+      Rails.logger.info "Processing batchSize: #{PER_PAGE_SIZE}, startIndex: #{start_index}"
+      response = service.work_processes(
+        batchSize: PER_PAGE_SIZE,
+        startIndex: start_index
+      )
+      parsed_response = response.parsed
+      occupation_standards = process_api_response(parsed_response)
+      occupation_standards.map(&:save)
```

Comment:
Do we need to worry about not doing any error checking on `save` here? This is going to silently fail if we are missing any validations.

Reply:
@littleforest staging server has a lot of invalid data and it may be fine for now. I agree, though: do you have any suggestion on what to do when we fetch an invalid record?

Reply:
If we're fine just skipping it, then this is fine and we can leave as-is. Otherwise, do we want to send the data to the error subscriber so we can investigate to see if there are other changes that either need to be made to the API, or will help us realize we aren't getting data from a correct field, for example?

Reply:
@littleforest I asked Patrick on Slack 😁